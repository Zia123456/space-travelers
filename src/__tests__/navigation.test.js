import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navigation from '../components/navigation';

it('renders correctly', () => {
  const navigation = renderer
    .create(<Router><Navigation /></Router>)
    .toJSON();
  expect(navigation).toMatchSnapshot();
});
