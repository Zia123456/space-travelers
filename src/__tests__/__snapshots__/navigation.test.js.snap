// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="nav-bar"
>
  <a
    className="nav-logo-link"
    href="#/"
  >
    <img
      alt="logo"
      className="nav-logo"
      src="logo.png"
    />
    <h1
      className="nav-logo-title"
    >
      Space Travelers' Hub
    </h1>
  </a>
  <ul
    className="nav-links"
  >
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
        style={
          Object {
            "textDecoration": "underline",
          }
        }
      >
        Rockets
      </a>
    </li>
    <li>
      <a
        className=""
        href="/Missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <div
      className="vertical-divider"
    />
    <li>
      <a
        className=""
        href="/MyProfile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </li>
  </ul>
</nav>
`;
